src/Nox.Reference.VatNumbers/Services/BrazilValidationService.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Nox.Reference.VatNumbers.Constants;
+using Nox.Reference.VatNumbers.Extension;
+using Nox.Reference.VatNumbers.Models;
+
+namespace Nox.Reference.VatNumbers.Services
+{
+    // Verifiend manually: FALSE
+    // Personal data cleaned: TRUE
+    internal class BrazilValidationService : IVatValidationService
+    {
+        // TODO: review
+        private const string _validationPattern = @"^\d{14}$";
```

Comment:
Could you introduce static file with regex expressions strings.

Reply:
I think it's better to keep them split per country for now. We can discuss later